sim/src/main/makefrag/firesim/Makefrag

```diff
@@ -241,7 +241,9 @@ endif
 disasm := 2>
 which_disasm := $(shell which spike-dasm 2> /dev/null)
 ifneq ($(which_disasm),)
-        disasm := 3>&1 1>&2 2>&3 | $(which_disasm) $(DISASM_EXTENSION) >
+    # Default to no extra spike-dasm arguments, Allowing cmdline addition
+    DISASM_EXTENSION ?=
+    disasm := 3>&1 1>&2 2>&3 | $(which_disasm) $(DISASM_EXTENSION) >
```

Comment:
Clarify that the default behavior is to run `spike-dasm` without any arguments but providing a variable that can be set on the cmdline.